test/odbc/test_pg_datatype_bit.cpp

```diff
@@ -0,0 +1,581 @@
+#include <gtest/gtest.h>
```

Comment:
Looks good. There are a few more test cases that were called out by Amazon as good to have:

> Determining if primary key or unique constraint exists on columns that support usage a the constraint type. Also test multi-column PK with mixed types like: int/smallint, int/varchar, int/datetime

Can these also be added, if possible?

Reply:
Is there a document I can reference for any additional test cases? 

For now I'll look into the one listed here after I finish up some other things.